code/modules/mob/abstract/new_player/new_player.dm

```diff
@@ -87,8 +87,28 @@ INITIALIZE_IMMEDIATE(/mob/abstract/new_player)
 			stat("Players: [totalPlayers]", "Players Ready: [totalPlayersReady]")
 			totalPlayers = 0
 			totalPlayersReady = 0
-			for(var/mob/abstract/new_player/player in player_list)
-				stat("[player.key]", (player.ready)?("(Playing)"):(null))
+
+			//Determine high job and display it next to "ready"
+			var/highjob
```

Comment:
This needs to be inside the first loop. Otherwise you'll have a leftover value from the previous iteration.